nana/modules/notes.py

```diff
@@ -82,7 +83,7 @@ async def save_note(_client, message):
     await message.edit(f'Saved note `{note_name}`!')
 
 
-@app.on_message(Filters.user(Owner) & Filters.command("get", Command))
+@app.on_message(filters.user(Owner) & filters.command("get", Command))
```

Comment:
Codacy found an issue: [get_note is too complex (17) (MC0001)](https://app.codacy.com/gh/MadeByThePinsHub/Nana-Remix/pullRequest?prid=6330175)